client/src/App.tsx

```diff
@@ -1,10 +1,14 @@
-import Sidebar from './components/Layout/Sidebar';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { routes } from './routes';
+
+const router = createBrowserRouter(routes);
 
 function App() {
   return (
     <div>
-      <Sidebar />
+      <RouterProvider router={router} />
     </div>
```

Comment:
Is the div element still needed? Moreover, you can add fallbackElement property to the RouterProvider with some kind of loader component or just for now string - "loading"